src/pages/Courses/components/FilterTable/FilterTable.jsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+
+import "./FilterTable.css";
+
+const filterData = [
+  {
+    heading: "Category",
+    name: "category",
+    inputType: "checkbox",
+    filterType: [
+      {
+        id: "marketing",
+        label: "Marketing",
+      },
+      {
+        id: "finance",
+        label: "Finance",
+      },
+      {
+        id: "technology",
+        label: "Technology",
+      },
+    ],
+  },
+  {
+    heading: "Ratings",
+    name: "rating",
+    inputType: "radio",
+    filterType: [
+      {
+        id: "four-and-above",
+        label: "4 stars and above",
+      },
+      {
+        id: "three-and-above",
+        label: "3 stars and above",
+      },
+      {
+        id: "two-and-above",
+        label: "2 stars and above",
+      },
+    ],
+  },
+  {
+    heading: "Sort By",
+    name: "sort",
+    inputType: "radio",
+    filterType: [
+      {
+        id: "high-to-low",
+        label: "Price - high to low ",
+      },
+      {
+        id: "low-to-high",
+        label: "Price - low to high",
+      },
+    ],
+  },
+];
+
+const FilterTable = () => {
+  return (
+    <aside className="filter-table">
+      <header>
+        <h1 className="h4">Filter</h1>
+        <button className="btn btn-link-primary">Clear</button>
+      </header>
+      <section className="price-group">
+        <h1 className="h5">Price</h1>
+        <form action="" className="price-form">
```

Comment:
Suggestion: action attribute is empty in here, could remove it if its not required.

Reply:
removed